includes/theme_recent_supporters.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @file
+ * Override recent supporters.
+ */
+
+/**
+ * Implements theme_recent_supporters().
+ *
+ * - Remove "clearfix" class on supporter list items.
+ * - Move wrapper outside so it’s also there for no supporters.
+ * - Add "no-bullet" class on list (Foundaton bullets won’t work when cycling).
+ * - Add classes indicating what supporter components to expect (comments, flags).
+ * - Put the “no activity” text in a paragraph.
+ *
+ * See recent_supporters/recent_supporters.module.
+ */
+function campaignion_foundation_recent_supporters($variables) {
+  $supporters = $variables['supporters'];
+  if ($show_country = $variables['show_country']) {
+    $countries = country_get_list();
+  }
+
+  $output = "<div id=\"{$variables['id']}\" class=\"recent-supporters-wrapper\">";
+
+  if (count($supporters) < 1) {
+    $output .= '<div class="no-activity-yet"><p>' . t('No recent activity.') . '</p></div>';
+    return $output;
+  }
+  else {
+    $classes = ['no-bullet', 'recent-supporters'];
+    $classes[] = $variables['all_actions'] ? 'all-actions' : 'single-action';
+    $classes[] = $variables['show_comment'] ? 'with-comments' : 'no-comments';
+    $classes[] = $show_country ? 'with-flags' : 'no-flags';
+    $output .= "<ul class=\"" . implode(' ', $classes) . "\">\n";
+    $count = 1;
+    foreach ($supporters as $supporter) {
+      $action_type = "";
+      if (!empty($supporter['action_type'])) {
+        $action_type = "action-type-" . $supporter['action_type'];
```

Comment:
Maybe use `drupal_clean_css_identifier()`...

Reply:
I didn’t touch that part of the function ;)